test/unit/model/qti/metadata/ontology/GenericLomOntologyClassificationInjectorTest.php

```diff
@@ -0,0 +1,531 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\test\unit\model\qti\metadata;
+
+use oat\generis\model\OntologyRdf;
+use oat\generis\test\TestCase;
+use core_kernel_classes_Class;
+use core_kernel_classes_ContainerCollection;
+use core_kernel_classes_Property;
+use core_kernel_classes_Resource;
+use oat\generis\model\data\Ontology;
+use oat\taoQtiItem\model\qti\metadata\MetadataInjectionException;
+use oat\taoQtiItem\model\qti\metadata\ontology\GenericLomOntologyClassificationInjector;
+use oat\taoQtiItem\model\qti\metadata\simple\SimpleMetadataValue;
+use PHPUnit\Framework\MockObject\MockObject;
+
+class GenericLomOntologyClassificationInjectorTest extends TestCase
```

Comment:
Thanks for the test! 👏  🚀

Reply:
For this case, we can extend TestCase from PhpUnit, we do not need the one from generis (which is deprecated).

Reply:
You are right, I missed that one. Changed in bbb92640837eca955413da978903b6307c29cce2, thanks! :)